src/app/course/course-practice/course-practice.component.scss

```diff
@@ -0,0 +1,3 @@
+.group {
+    max-width: 30.25rem;
```

Comment:
I dont see this being used, please remove or if it is used use tail wind instead.